### DISCUSSION TITLE
Does `Diagnostics.UnusedIncludes` only work for quoted but not bracketed `#include`s?

### DISCUSSION THREAD

**Discussion by metaleap** [clangd/clangd#1855] [Q&A] [answered]
This is in both my project's `.clang` and my `~/.config/clangd/config.yaml`:

```yaml
Diagnostics:
  UnusedIncludes: Strict
```

But trying it out across the codebase, I notice warnings only for `#include "..."`-patterned but not `#include <...>`-patterned header includes.

Is this by design? For the `<...>` case the build-duration aspect might matter less but the tidiness aspect still fully does.

**Comment by HighCommander4** [comment, marked as answer]
> But trying it out across the codebase, I notice warnings only for `#include "..."`-patterned but not `#include <...>`-patterned header includes.
> 
> Is this by design?

Based on #1774, it currently seems to be by design.

Please feel free to weigh in on that issue if you'd like to see a change to this behaviour.
